src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -440,23 +497,32 @@ void activity_classifier::perform_training_iteration() {
     float batch_loss = std::accumulate(loss_batch.data(),
                                        loss_batch.data() + loss_batch.size(),
                                        0.f, std::plus<float>());
+
+    
     cumulative_batch_loss += batch_loss / batch.batch_info.size();
 
+    
+    const shared_float_array& output = results.at("output");
+    float cumulative_per_batch_accuracy = 0.f;
+    cumulative_per_batch_accuracy = get_accuracy_per_batch(prediction_window, num_classes, output, batch);
```

Comment:
Is this right? I don't see `cumulative_per_batch_accuracy` being used anywhere. Do we still need this variable?